### ISSUE TITLE
Edit Modal Not Working

### ISSUE THREAD

**Issue by abhinav-bohra** [KasperOlesen/DataTable-AltEditor#117]
When I select a row and click on edit, the Edit modal pop up comes. It displays column names correctly but it does not display field values. 
Instead of holding current value, the fields store nothing (i.e. they are blank).

**Comment by luca-vercelli** [comment]
This is not true in the online examples (see e.g. https://luca-vercelli.github.io/DataTable-AltEditor/example/02_in_memory_objects/example2.html)

Can you provide some more details, or some jsfiddle?

**Comment by abhinav-bohra** [comment]
> 
> 
> This is not true in the online examples (see e.g. https://luca-vercelli.github.io/DataTable-AltEditor/example/02_in_memory_objects/example2.html)
> 
> Can you provide some more details, or some jsfiddle?

**Comment by abhinav-bohra** [comment]
Yes, you are right, @luca-vercelli.
It is working perfectly in examples. However, it wasn't working as expected when I was using it in my project. I have fixed it now.

I was loading data from a local JSON file. The problem was that my "data" label in "columnDefs" had space in the name. I changed it by using underscores and it works.

Original Format :
var columnDefs = [
              { data: "Category ID", title: "ID",},
              { data: "Category Name", title: "Name",}              
 ];

NewFormat (This works):
var columnDefs = [
              { data: "Category_ID", title: "ID",},
              { data: "Category_Name", title: "Name",}              
 ];

Thanks a lot for the plugin. Great Work!!!
